commander/src/commands/transaction/create/delegate.ts

```diff
@@ -111,29 +109,28 @@ export default class DelegateCommand extends BaseCommand {
 			throw new ValidationError('Enter a valid fee in number string format.');
 		}
 
-		const processFunction = processInputs(
-			nonce,
-			normalizedFee,
-			networkIdentifier,
-			username,
-		);
-
 		if (noSignature) {
-			const noSignatureResult = processFunction({
-				passphrase: undefined,
-			});
+			const noSignatureResult = processInputs(
+				nonce,
+				normalizedFee,
+				networkIdentifier,
+				username,
+				undefined,
```

Comment:
`undefined` is required to pass? if the passphrase is not sent it will be undefined know?